package_manager/dpkg_parser.py

```diff
@@ -17,22 +17,34 @@
 import io
```

Comment:
It looks like we're not running pylint on this file, could you do that manually and open a bug to get it in travis?